ASIM/dev/ASimTester.csv

```diff
@@ -0,0 +1,1377 @@
+ColumnName,ColumnType,Class,Schema,LogicalType,ListOfValues,Aliased

```

Comment:
incorrect location, please remove this change

Reply:
Please confirm what exactly required to be changed here.

Reply:
remove adding this file. The correct ASIMTester.csv is already being updated.

Reply:
changes done.